.github/workflows/sbom_generator.yaml

```diff
@@ -0,0 +1,73 @@
+# Generate SBOM for latest version of dotCMS and put into core-test-repo
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      dotcms_version:
+        description: 'The dotCMS version tag (e.g., v24.07.30)'
+        required: true
+        default: 'v24.07.30'
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write  # Ensure write access to contents
+
+    steps:
+      - name: Checkout core-test-results repository
+        uses: actions/checkout@v3
+        with:
+          repository: dotCMS/core-test-results
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
So it works w/ github token? no need for PAT anymore?

Reply:
@spbolton, @vicozizou, and @dcolina have done a great job advancing our practice here.  It's reccommended to use the `GITHUB_TOKEN` value so that we don't hardcode credentials.  Using the `GITHUB_TOKEN` solution evaluates to the github user that invoked the action.

Today's best practice is to move to full blown github app, but we've not started that migration yet.

Reply:
the _catch_ here is that the invoking user needs enough privileges on the repos.  In this case probably [core](https://github.com/dotcms/core) and [core-test-results](https://github.com/dotcms/core-test-results)